projects/sd-jwt-js.md

```diff
@@ -0,0 +1,80 @@
+# Project Name
+
+SD-JWT Javascript Reference Implementation
+
+# Project Description
+
+This is the reference implmentation of [IETF SD-JWT specification](https://datatracker.ietf.org/doc/draft-ietf-oauth-selective-disclosure-jwt/) in Javascript.
+
+Hopae, a founding member of OpenWallet Foundation, is building wallet module in Javascript and need this project as a core component.
+
+# Alignment with the OpenWallet Foundation Mission
+
+This library is for the Selective Disclosure capability which is one of the important building blocks of digital identity wallets.
+
+# Current Code of Conduct
+
+none
+
+# TAC Sponsor
+
+none
+
+# Project License
+
+Apache 2.0
+
+# Source Control
+
+none
+
+# Issue Tracker
+
+none
+
+# External Dependencies
+
+TBD
```

Comment:
Should we wait for the dependencies list before evaluating this proposal, or does this `TBD` refer to the future?

Are we aware of the licenses of these dependencies and their compatibility or possible collision with Apache 2.0?